### PR TITLE
Adding type for geo json

### PR DESCRIPTION
**Pull request by ViBiOh**
Fixing #24 

# Before creating my pull-request, I ensure:

- [x] Javascript documentation is written
- [x] Unit tests are written and green
- [x] I've cleaned my WIP or meaningless commits
- [x] Examples and documentation site is up-to-date

### CODE REVIEW

File: src/Helpers/index.js [hunk 1]
```diff
@@ -32,9 +32,9 @@ const CIRCLE_POINTS_CONFIG = 64;
 export const UNITS = ['kilometers', 'meters', 'miles', 'feet'];
 
 /**
- * Convert the radius into the unit given
- * @param {Number} radius unit
- * @param {String} unit to convert to if necessary
+ * Convert the radius on given unit to a compatible unit for turf/circle
+ * @param {Number} radius on given unit
+ * @param {String} unit of radius given
```
Comment (danousna) [inline src/Helpers/index.js]: (Nitpicking)
I would suggest something closer to this :
```
* Convert the given radius into the given unit and returns a turf/circle compatible object.
* @param {Number} radius of given unit
* @param {String} unit of given radius
```

File: src/Helpers/index.js [hunk 2]
```diff
@@ -171,7 +174,7 @@ export function drawGeoJSON(map, id, data, paint = {}, onClick) {
 
         map.addLayer({
             id: layerId,
-            type: 'fill',
+            type,
```
Comment (danousna) [inline src/Helpers/index.js]: Will mapbox throw some kind of error if the type is un-recognized ? Should we try catch or validate the type ? I didn't find anything in the mapbox docs regarding this.

File: src/Helpers/index.js (hunk 1 above)
Comment (ViBiOh) [reply src/Helpers/index.js]: Not exactly, the radius is not *converted* into the given unit, the radius is *assumed* to be in the given unit and converted into compatible unit for turf/cicle

File: src/Helpers/index.js (hunk 2 above)
Comment (ViBiOh) [reply src/Helpers/index.js]: It's define [here](https://docs.mapbox.com/mapbox-gl-js/style-spec/#layers) but I think it's better to keep it as a wrapper and not starting to duplicate checks made in the underlying library.

**Pull request merged**